streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -3636,9 +3586,9 @@ public void shouldCommitViaConsumerIfEosDisabled() {
     @Test
     public void shouldCommitViaProducerIfEosAlphaEnabled() {
         final StreamsProducer producer = EasyMock.mock(StreamsProducer.class);
-        expect(activeTaskCreator.streamsProducerForTask(anyObject(TaskId.class)))
-            .andReturn(producer)
-            .andReturn(producer);
+        when(activeTaskCreator.streamsProducerForTask(any(TaskId.class)))
+                .thenReturn(producer)
+                .thenReturn(producer);
```

Comment:
```suggestion
```
You do not need to specify this multiple times. If you specify it once Mockito should take care of continuing to return the same mock on every invocation.

Reply:
updated, thank you.